dino.icu.yaml

```
@@ -52,6 +52,11 @@
 #   type: CNAME
 #   value: cname.vercel-dns.com.
 
+archives:
+  - ttl: 600
+    type: CNAME
+    value: crabby605.github.io
```

Don't forget the trailing dot!
```suggestion
    value: crabby605.github.io.
```

---

@crabby605 This still needs to be resolved before this can be merged.